src/components/Navbar.jsx

```diff
@@ -1,44 +1,44 @@
-import { useState } from 'react';
-import * as FaIcons from 'react-icons/fa';
-import * as AiIcons from 'react-icons/ai';
-import { Link } from 'react-router-dom';
-import { SidebarData } from './SidebarData';
-import './Navbar.css';
-import { IconContext } from 'react-icons';
+import UserProfile from "@/components/document/Header/UserProfile";
 
-function Navbar() {
-  const [sidebar, setSidebar] = useState(false);
+const NavbarSubSection = ({ subsectionName }) => {
+  return <div 
+    className='px-3 p-2 hover:bg-[#5C00B8] rounded'
+    onClick={(e) => {
+      print(e)
+      // TODO: Do some routing here
+    }}>
+    {subsectionName}
+  </div>
+}
 
-  const showSidebar = () => setSidebar(!sidebar);
+const NavbarSection = ({ title, subsections }) => {
+  return <div className=' text-lg font-bold py-3'>
+    {title}
+    <div className='ml-3 my-2 text-base font-light'>
+      <NavbarSubSection subsectionName={subsections[0]}/>
+      <NavbarSubSection subsectionName={subsections[1]}/>
+    </div>
+  </div>
 
+}
+
+function Navbar() {
   return (
     <>
-      <IconContext.Provider value={{ color: '#fff' }}>
-        <div className='navbar'>
-          <Link to='#' className='menu-bars'>
-            <FaIcons.FaBars onClick={showSidebar} />
-          </Link>
+      <div className='fixed w-[350px] h-screen bg-[#340068] text-white'>
+        <div className='grid grid-cols-1 divide-y divide-gray-400 m-[30px]'>
+          <div className='h-[80px] text-2xl font-bold'>
+            [redacted]
+          </div>
+          <NavbarSection title={"My Documents"} subsections={["To Redact", "In Progress"]}/>
```

Comment:
Another code style nitpick but It might be cleaner to store the title and subsections (and also links) in a constant object array, and then populate this with a for loop. So something like
```js
const sections = [
    { 
        title: "My Documents",
        subsections: [
            {
                name: "To Redact",
                link: "blah",
            }, {
                name: "In Progress",
                link: "blah"
            },

        ]
    }
    // and so on
] 
```